clang/test/SemaSYCL/no-prototype-func.c

```diff
@@ -0,0 +1,3 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -emit-llvm -verify -fms-extensions %s
```

Comment:
Why `fms-extensions` flag is required?

Reply:
This command-line option was passed while trying to compile on windows. It had something to do with MSVC++ compiler. I will remove this.